index.css

```
@@ -0,0 +1,171 @@
+html body {
+    font-family: Helvetica, sans-serif;
+    font-size: 22px;
+    margin: 0;
+    padding: 0;
+    background-color: black;
+    color: rgba(255, 246, 240, 0.9);
+}
+
+nav {
+    display: inline-flex;
```

Fixed is an option, but not the best choice in this use-case. Because its fixed, you then need to spaced the below image to make up for the extra `70px`. I would reccomend using `sticky`. If you are unfamiliar with `sticky`, these are the [docs](https://developer.mozilla.org/en-US/docs/Web/CSS/position). 
```suggestion
  position: sticky;
```